### DISCUSSION TITLE
Pinecone, Langchain and Namespaces

### DISCUSSION THREAD

**Discussion by marius-smg** [FlowiseAI/Flowise#537] [General]
Hey, yesterday Pinecone announced that it will no longer support namespaces on free starter plans. In their forums someone said that Langchain has abandoned namepsaces as parameters. Does this also affect flowise?

**Comment by chungyau97** [comment]
Hi @marius-smg, 

We do receive people failing to use Pinecone as their vector store.
In our [Discord Community](https://discord.gg/xZ63HhJn), their solution to it is using the paid plan.
